docs/rfc/0003-emoji-gitlog.md

```diff
@@ -0,0 +1,70 @@
+# 0003. Emoji git log
+
+<!-- 
+    Date representing when the RFC was submittet for review 
+
+    Example:
+    Date: 08.02.2021
+-->
+Date: 15.02.2021
+
+## Context
+
+<!--
+    Short description of why this decision is needed
+
+    Example:
+    We want to better document our decisions
+-->
+
+Contributors should be able to easily create meaningful commit messages.
+
+## Decision
```

Comment:
This section describes the git format lag nicely, however it doesn't state what the actual decision is. Is this optional? Should we do it for some commits, most commits, or all?

I think the git log would look nice if we follow this RFC. However, it might be too intrusive to enforce it (require it from all/most commits), as it requires the developer to either set up some [helpers](https://github.com/ahmadawais/Emoji-Log#bashzsh-workflow) or somehow lookup emojis to do commits. So in my (current) opinion this should be optional.

Reply:
Yeah. I left that part out because I wanted some opinions. Mostly I just wanted to formalize my system and show it to y'all

Reply:
But yes, good point. I've yet to get a hang of writing RFC's